Engine/Setup/ConsoleSetupHandler.cs

```diff
@@ -176,6 +180,23 @@ public bool Setup(IAlgorithm algorithm, IBrokerage brokerage, AlgorithmNodePacke
         }
 
         /// <summary>
+        /// Get the available data feeds from config.json,
+        /// If none available, throw an error
+        /// </summary>
+        private static Dictionary<SecurityType, List<TickType>> GetConfiguredDataFeeds()
```

Comment:
This method exists in multiple places. Consider moving it to a common location to reduce code duplication.